exercises/queen-attack/.meta/src/reference/java/Queen.java

```diff
@@ -0,0 +1,39 @@
+final class Queen {
+
+    private final int row;
+
+    private final int column;
+
+    Queen(final int row, final int column) throws IllegalArgumentException {
```

Comment:
The `throws IllegalArgumentException` isn't needed since `IllegalArgumentException` is an [unchecked exception](https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html). I know this isn't your fault since it was like this before, so you don't have to fix it, we can open a separate issue for it. But if you do feel like fixing it now then that would be great 🙂